packages/fx-core/src/plugins/resource/apiconnector/utils.ts

```diff
@@ -2,13 +2,27 @@
 // Licensed under the MIT license.
 "use strict";
 import * as fs from "fs-extra";
-import { LanguageType, FileType } from "./constants";
+import { LanguageType, FileType, AuthType } from "./constants";
 export interface ApiConnectorConfiguration extends Record<any, any> {
   ComponentPath: string[];
   APIName: string;
-  ApiAuthType?: string;
   EndPoint: string;
   ApiUserName?: string;
+  AuthConfig: AuthConfig;
+}
+
+export interface AuthConfig {
```

Comment:
can we move the interfaces to a standalone data model folder or ts file?

Reply:
It is fixed. Thanks.